### ISSUE TITLE
[bug] TypeError: Object of type TIT2 is not JSON serializable

### ISSUE THREAD

**Issue by skj-skj** [iiPythonx/lrcup#3]
failed to publish with the following error
```
raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type TIT2 is not JSON serializable
```

<details>
  <summary>Full Traceback</summary>

  ```
  Traceback (most recent call last):
  File "/data/data/com.termux/files/usr/bin/lrcup", line 8, in <module>
    sys.exit(lrcup())
             ^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/lrcup/__main__.py", line 107, in upload
    success = lrclib.publish(
              ^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/lrcup/controller.py", line 87, in publish
    return self.request(
           ^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/lrcup/controller.py", line 22, in request
    return getattr(self.session, method)(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/requests/sessions.py", line 575, in request
    prep = self.prepare_request(req)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/requests/sessions.py", line 484, in prepare_request
    p.prepare(
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/requests/models.py", line 370, in prepare
    self.prepare_body(data, files, json)
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/requests/models.py", line 510, in prepare_body
    body = complexjson.dumps(json, allow_nan=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/json/encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/json/encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/json/encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type TIT2 is not JSON serializable
  ```
</details>

**Comment by iiPythonx** [comment]
Fixed in 85cd12d7cd1e699eb56f4b6cb4da9c1819f4ab3d, I need to stop pushing changes without testing first :)
If you find any other issues with it, lmk but it should be fixed at this point *(hopefully)*.

0.5.2 should be published soon when GitHub actions runs.

**Comment by skj-skj** [comment]
thanks for the quick fix, I am able to publish the lyrics to lrclib.
